plugin/geoip/README.md

```diff
@@ -47,7 +47,7 @@ geoip [DBFILE] {
 ```
 
 * **DBFILE** the mmdb database file path.
-* `edns-subnet`: Optional. Use [EDNS0 subnet](https://en.wikipedia.org/wiki/EDNS_Client_Subnet) (if present) for Geo IP instead of the source IP of the DNS request. This helps with identifying the closest source IP address as well as making GeoIP testing easy: `dig +subnet=1.2.3.4 @dns-server.example.com www.geo-aware.com`.
+* `edns-subnet`: Optional. Use [EDNS0 subnet](https://en.wikipedia.org/wiki/EDNS_Client_Subnet) (if present) for Geo IP instead of the source IP of the DNS request. This helps identifying the closest source IP address through intermediary DNS resolvers, and it also makes GeoIP testing easy: `dig +subnet=1.2.3.4 @dns-server.example.com www.geo-aware.com`.
```

Comment:
I think "intermediary DNS resolvers" might be technically imperfect here  - _but I think its fine as is_.  The specific situation that ends-subnet aims to assist is when any intermediate network "stuff" changes the source ip address of the request packet in transit, thus obfuscating the original client's IP.  More specifically, when the new IP address is not "near" the client.

Reply:
I agree, it's a nice 80% solution :) I'm fine with that though.